javatests/arcs/core/storage/ReferenceModeStoreTest.kt

```diff
@@ -700,6 +700,32 @@ class ReferenceModeStoreTest {
     assertThat(store.backingStore.stores.size).isEqualTo(0)
   }
 
+  @Test
+  fun clearsbacking_andcontainer_stores() = runBlockingTest {
```

Comment:
Nit: Test names usually follow this structure (and capitalisation):

```
fun methodUnderTest_conditionDetails_expectedOutcome()
```

So here (where there's no relevant conditions), I'd suggest just: `close_closesBackingAndContainerStores`

It's also debatable whether this should in fact be two tests, `close_closesBackingStores` and `close_closesContainerStore`, but that's possibly overkill.

Reply:
I stuck with the one test just for simplicity, but renamed it.